src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,510 @@
+		const breadcrumbs = this.$slots.default || []
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		// The array of all created VNodes
+		const crumbs = []
+		/**
+		 * We show the first half of the breadcrumbs before the Actions dropdown menu
+		 * which shows the hidden breadcrumbs.
+		 */
+		const crumbs1 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		// Add the breadcrumbs to the array of the created VNodes, check if hiding them is necessary.
+		this.addCrumbs(crumbs, crumbs1)
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			// Create the Actions menu
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement('Actions', {
+					class: 'dropdown',
+					// We want the Actions to always show as a dropdown menu
+					props: {
+						forceMenu: true,
+					},
+					// Add a ref to the Actions menu
+					ref: 'actions',
+					// Add handlers so the Actions menu opens on hover
+					nativeOn: {
+						dragstart: this.dragStart,
+						dragenter: () => { this.$refs.actions.opened = true },
+						dragleave: this.closeActions,
+					},
+				// Add all hidden breadcrumbs as ActionRouter or ActionLink
+				}, this.hiddenCrumbs.map(crumb => {
+					// Get the parameters from the breadcrumb component props
+					const to = crumb.componentOptions.propsData.to
+					const href = crumb.componentOptions.propsData.href
+					const disabled = crumb.componentOptions.propsData.disableDrop
+					// Decide whether to show the breadcrumbs as ActionRouter or ActionLink
+					let element = 'ActionLink'
+					let path = href
+					if (to) {
+						element = 'ActionRouter'
+						path = to
+					}
+					return createElement(element, {
+						class: 'crumb',
+						props: {
+							to: to,
+							href: href,
+							icon: 'icon-folder',
+						},
+						// Prevent the breadcrumbs from being draggable
+						attrs: {
+							draggable: false,
+						},
+						// Add the drag and drop handlers
+						nativeOn: {
+							dragstart: this.dragStart,
+							drop: ($event) => this.dropped($event, path, disabled),
+							dragover: this.dragOver,
+							dragenter: ($event) => this.dragEnter($event, disabled),
+							dragleave: ($event) => this.dragLeave($event, disabled),
+						},
+					},
+					crumb.componentOptions.propsData.title
+					)
+				}))]
+			))
+		}
+		// The second half of the breadcrumbs
+		const crumbs2 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(Math.round(breadcrumbs.length / 2))
+			: []
+		this.addCrumbs(crumbs, crumbs2)
+
+		return createElement('div', { class: 'breadcrumb', ref: 'container' }, crumbs)
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../fonts/scss/iconfont-vue';
+
+.breadcrumb {
+	width: 100%;
+	flex-grow: 1;
+
+	div.crumb {
```

Comment:
This whole section should go into the Breadcrumb component, no?

Reply:
Yeah, you are right, this can go into into the Breadcrumb component.

Reply:
Adjusted it now.